spectrocloud/schemas/cluster_location_config.go

```diff
@@ -0,0 +1,42 @@
+package schemas
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func ClusterLocationSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"country_code": {
+					Type:     schema.TypeString,
```

Comment:
```suggestion
					Type:     schema.TypeString,
					Description: "The country code of the country the cluster is located in.",
```

@nikchern do you know what format the country code should be in? ISO 3166? If we can specify the ISO, that would help a lot.